skore/src/skore/sklearn/_comparison/utils.py

```diff
@@ -292,7 +293,7 @@ def sort_by_split(df: pd.DataFrame) -> pd.DataFrame:
 
         return df
 
-    results = individual_results.copy()
+    results = copy.deepcopy(individual_results)
```

Comment:
```suggestion
    # trigger a deepcopy of the underlying dataframe to avoid
    # to mutate them if they are provided by the cache
    results = copy.deepcopy(individual_results)
```